org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/snippets/ProcessingInstructionSnippetContext.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+* Copyright (c) 2020 Balduin Landolt and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
```

Comment:
Please add 

```
 *
 * SPDX-License-Identifier: EPL-2.0
 *
```

Reply:
For the headerI used the vscode snippet (as defined here: https://github.com/eclipse/lemminx/blob/master/.vscode/project.code-snippets). I guess that should be added there as well, then?